MediaBrowser.sln

```diff
@@ -41,8 +41,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Emby.Naming", "Emby.Naming\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Emby.XmlTv", "Emby.XmlTv\Emby.XmlTv\Emby.XmlTv.csproj", "{6EAFA7F0-8A82-49E6-B2FA-086C5CAEA95B}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IsoMounter", "Emby.IsoMounting\IsoMounter\IsoMounter.csproj", "{9BA471D2-6DB9-4DBF-B3A0-9FB3171F94A6}"
```

Comment:
If this a artifact from the PR that moved this to a plugin? Is this supposed to be in this PR?

Reply:
Apparently this wasn’t changed in that PR, so current master is failing. This could be a separate PR. 🤷‍♂️

Reply:
Mmm might as well have it in this one. It's a "warning" right?

Reply:
It’s a build failure, when trying to build the sln, building the startup project works fine.